packages/blockchain-api/src/utils.ts

```diff
@@ -40,3 +43,52 @@ export function formatCommentString(functionCallHex: string): string {
 export function formatDateString(date: Date) {
   return date.toISOString().split('T')[0]
 }
+
+let goldTokenAddress: string
+let stableTokenAddress: string
+let attestationsAddress: string
+let tokenAddressMapping: { [key: string]: string }
+export async function getContractAddresses() {
+  if (goldTokenAddress && stableTokenAddress && attestationsAddress) {
+    console.info('Already got token addresses')
+    return { tokenAddressMapping, attestationsAddress }
+  } else {
```

Comment:
Nit: No need for this else, the if statement returns
See here:
https://stackoverflow.com/questions/268132/invert-if-statement-to-reduce-nesting